target/linux/lantiq/image/vr9.mk

```diff
@@ -147,6 +147,38 @@ define Device/avm_fritz3390
 endef
 TARGET_DEVICES += avm_fritz3390
 
+define Device/avm_fritz5490
+  $(Device/dsa-migration)
+  $(Device/AVM)
+  $(Device/NAND)
+  DEVICE_MODEL := FRITZ!Box 5490/5491
```

Comment:
Please use this instead:
````
  DEVICE_MODEL := FRITZ!Box 5490
  DEVICE_ALT0_VENDOR := AVM
  DEVICE_ALT0_MODEL := FRITZ!Box 5491
````

It it possible to later also support both FRITZ!Box 5490 and FRITZ!Box 5491 with the same image? 
If they later need different device trees please already create two different dts files now to make the upgrade easier in the future.